scipy/stats/_qmc.py

```diff
@@ -1244,7 +1297,7 @@ def _standard_normal_samples(self, n=1):
         if self._inv_transform:
             # apply inverse transform
             # (values to close to 0/1 result in inf values)
-            return stats.norm.ppf(0.5 + (1 - 1e-10) * (samples - 0.5))
+            return stats.norm.ppf(0.5 + (1 - 1e-10) * (samples - 0.5)) # type: ignore[attr-defined]
```

Comment:
Had to add this because of python/mypy#7182. Otherwise, I had this error : 
`scipy/stats/_qmc.py:1300: error: Module has no attribute "norm"  [attr-defined]`